Defend The Divine/Assets/Scripts/Spells/SpellActivate.cs

```diff
@@ -24,6 +24,9 @@ void Start()
 
     public void OnClick(int index)
     {
+        // Trigger camera shake
```

Comment:
Can you drag in a "ShakeBehavior" reference and call it that way, or even set it to a variable in Start?
I know spells aren't called that often, but i feel searching for it each time isn't very good